mutable_tree.go

```diff
@@ -712,10 +718,15 @@ func (tree *MutableTree) enableFastStorageAndCommit() error {
 
 	itr := NewIterator(nil, nil, true, tree.ImmutableTree)
 	defer itr.Close()
+	var upgradedFastNodes uint64
 	for ; itr.Valid(); itr.Next() {
+		upgradedFastNodes++
 		if err = tree.ndb.SaveFastNodeNoCache(NewFastNode(itr.Key(), itr.Value(), tree.version)); err != nil {
 			return err
 		}
+		if upgradedFastNodes%commitGap == 0 {
+			tree.ndb.Commit()
```

Comment:
why doing a commit here?

Reply:
when we update `commitGap` fastNodes, we commit it, or when we met a huge IAVL tree, the memory cost will be huge too.